### ISSUE TITLE
Versioning of FreeMarker

### ISSUE THREAD

**Issue by krisaju95** [kaola-fed/freemarker#18]
Not exactly an issue but just a missing piece of information in the documentation. Could you please add the [version](https://freemarker.apache.org/docs/app_versions.html) of the FreeMarker that this tool currently supports with each release. Or at least what the current version is in case you guys aren't planning to continue supporting the project.

Great work btw! Just used this for a sandbox app I was working on.

**Comment by zhangwx26** [comment]
mark

**Comment by lucascudo** [comment]
Yes, and please accept this PR, which updates the supported version: https://github.com/kaola-fed/freemarker/pull/20
